TrackletAlgorithm/TrackHandler.h

```diff
@@ -0,0 +1,50 @@
+#ifndef TrackletAlgorithm_TrackHandler_h
+#define TrackletAlgorithm_TrackHandler_h
+#include "TrackFitMemory.h"
+#include "hls_stream.h"
+const int NBarrelStub = 4;
+const int NDiskStub = 4;
```

Comment:
I think some `typedefs` would improve the readability of the code throughout. At least an alias for `TrackFit<NBarrelStub, NDiskStub>`, and maybe others that appear frequently. Not strictly necessary, but I think some of the long lines would be easier to digest.

Reply:
Sure, will implement those!